remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/AppRemoteMessagingRepository.kt

```diff
@@ -58,8 +58,9 @@ class AppRemoteMessagingRepository(
     override fun messageFlow(): Flow<RemoteMessage?> {
         // Experiment: Ask for Default Browser More Than Once
         val daysSinceInstalled = userBrowserProperties.daysSinceInstalled()
+        val isDefaultBrowser = userBrowserProperties.defaultBrowser()
```

Comment:
I'm fine hardcoding experiment specific logic here, but can we do it only the experiment variant. Same goes for the logic that was added in the first release. Otherwise people on this app version outside experiment, will never get a default message if `daysSinceInstalled` different to 0, 1 or 2.